homeassistant/components/light/hue.py

```diff
@@ -27,6 +29,10 @@
 
 PHUE_CONFIG_FILE = "phue.conf"
 
+SUPPORT_HUE = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR_TEMP | SUPPORT_EFFECT |
```

Comment:
It's not the end of the world when features are being mis-represented. We will have time to get PRs in to get this fixed before the next release.